pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -323,6 +337,11 @@ func (cm *containerManagerImpl) Start() error {
 				}
 			}
 		}
+		for _, task := range cm.periodicTasks {
```

Comment:
Do we want these scheduled in serial, or is it better to schedule them in separate go routines?